build.gradle

```diff
@@ -1,14 +1,14 @@
 buildscript {
     repositories {
-        mavenCentral()
+        jcenter()
         maven {
             url 'https://dl.bintray.com/kotlin/kotlin-eap'
         }
         maven {
             url 'http://oss.sonatype.org/content/repositories/snapshots'
```

Comment:
Do we actually needs these two at this point?

Reply:
Let's try to remove them, I hope we don't use snapshots nor eap for public releases